rocker-compiler/src/main/java/com/fizzed/rocker/compiler/JavaGenerator.java

```diff
@@ -1053,6 +1053,80 @@ else if (unit instanceof SwitchDefaultBlockEnd) {
                         .append(blockEnd.pop())
                         .append(" // default end ").append(sourceRef(unit)).append(CRLF);
             }
+
+            else if(unit instanceof SwitchExpressionBlock){
+
+                SwitchExpressionBlock block = (SwitchExpressionBlock) unit;
+
+                // break support via try and catch mechanism (works across lambdas!)
+                tab(w, depth+indent)
+                        .append("try {").append(CRLF);
+
+                depth++;
+
+                tab(w, depth+indent)
+                        .append("switch ")
+                        .append(block.getExpression())
+                        .append(" {").append(CRLF);
+
+                blockEnd.push("}");
+
+                depth++;
+            } else if (unit instanceof SwitchExpressionBlockEnd) {
+                depth--;
+
+                tab(w, depth+indent)
+                        .append(blockEnd.pop())
+                        .append(" // switch end ").append(sourceRef(unit)).append(CRLF);
+
+                depth--;
+
+                // break support via try and catch mechanism (works across lambdas!)
```

Comment:
Break support should be removed for switch expressions -- someone could actually accidentally include one and it'd work with the try/catch you included here (copied from the standard switches)

Reply:
done